docs/layers/js-tutorial.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Tutorial
+sidebar_label: Example in Javascript
+slug: /layers/js-tutorial
+---
+
+# Using a Layer in Javascript
+
+This example shows how to setup a Layer with multiple parameters and then run multiple experiments with different combinations of those parameters.
+
+## Create a Layer
+
+First, create a new layer by logging into the [Statsig Console](https://console.statsig.com), navigate to Experiments, select the Layers tab and then select Create New.
+
+![Screen Shot 2022-03-21 at 8 33 56 AM](https://user-images.githubusercontent.com/95646168/159297814-db7a8423-ecc5-4370-8b41-733e97954a55.png)
+
+In the dialog that appears, give your new Layer a name and then select Create. 
+
+![Screen Shot 2022-03-21 at 8 46 15 AM](https://user-images.githubusercontent.com/95646168/159298510-ef27ef28-fc2b-4301-8156-df6fb6703586.png)
+
+
+With this Layer created, you can now add parameters that can be shared across experiments.
+
+
+## Add Parameters
+
+On the Layer page, select Add Parameter. 
+
+![Screen Shot 2022-03-21 at 10 13 30 AM](https://user-images.githubusercontent.com/95646168/159327580-837ab0ef-5449-4cb7-b09b-41f3bfa93e63.png)
+
+
+In the dialog that appears, you can enter the Name, Type and Default Value for your parameter.
+
+![Screen Shot 2022-03-21 at 10 13 51 AM](https://user-images.githubusercontent.com/95646168/159327612-b0f09c4c-9d47-48d0-972b-e1d27bc3d3f2.png)
+
+
+## Setup Experiments
+
+We can now reference our Layer during Experiment creation, allowing us to pull in these parameters and share them across Experiments.
+
+When creating and Experiment, expand the Advanced options, check Add Layer, and select your Layer from the layer dropdown.
```

Comment:
spelling nit: "When creating an Experiment"

Reply:
Also I added a new flow for this; on the layer's page, you can click "Create new Experiment in Layer". That avoids them having to go into the Advanced section and find their layer.

Fine with leaving docs like this though, just an FYI